lib/fetchContent.js

```diff
@@ -301,9 +303,177 @@ function getArticle(uuid) {
 		});
 }
 
+async function getRecentArticles(days){
+	try{
+		const date = time.getDatetimeRange('days', days, 0);
+		const params = {
+			queryString: "lastPublishDateTime:>" + date.next,
+			maxResults : 100,
+			aspects : [ 
+				"audioVisual",
+				"editorial",
+				"images",
+				"lifecycle",
+				"location",
+				"master",
+				"metadata",
+				"nature",
+				"provenance",
+				"summary",
+				"title"
+			],
+			facets : {"names":[ "organisations", "organisationsId", "people", "peopleId", "topics", "topicsId"], "maxElements":-1}
```

Comment:
would it be helpful to have the list of facets either as env vars or UI selection?

Reply:
Done 😃